release-team/release-team-onboarding.md

```diff
@@ -0,0 +1,65 @@
+# Release Team Onboarding Guide
+
+When joining the Release Team or taking on a new role on the Release Team, there are various lists and groups you'll need to be apart of, various permissions that you'll need to have, keys you may need to know, etc. For roles ranging from someone new to the Kubernetes Community to the incoming Release Lead, this document aims to enumerate the access that you'll need to ensure you have to various repos and tools.
+
+## New Members
+
+If you're a brand new member to the Kubernetes Community or the Release Team, welcome! It's great to have you involved with SIG Release.
+
+### Slack
+
+The communication and coordination for the overwhelming majority of SIG Release topics happens in the [`#sig-release`](https://kubernetes.slack.com/messages/C2C40FMNF/) channel of the Kubernetes Slack. If you need access to the Kubernetes Slack, please see [http://slack.k8s.io/](http://slack.k8s.io/). If you need help with any of your additional access needs, `#sig-release` is a good place to ask for help.
+
+### Kubernetes Community Membership
+
+The Kubernetes Community defines Members as the following:
```

Comment:
non-blocking suggestion: maybe "defines" should link to the definition?
I do see we link below, but it might be nice to clarify where we claim to be pulling the definition from, especially if/when it goes stale later.